
<launch>
  <!-- <arg name="audio_path" default = "$(find xf_mic_asr_offline)" /> -->

  <rosparam command="load" file="$(find xf_mic_asr_offline)/config/recognition_params.yaml" />

  <!-- 录音调用控制器，包含休眠功能 -->
  <node pkg="xf_mic_asr_offline" type="call_recognition" name="call_recognition" />

  <!-- 命令控制器，命令词识别结果转化为对应的执行动作 -->
  <node pkg="xf_mic_asr_offline" type="command_recognition" name="command_recognition" output="screen">
    <param name="audio_path" type="string" value="$(find xf_mic_asr_offline)"/>  
  </node>

  <!-- 功能节点状态反馈，用于状态语音播报 -->
  <node pkg="xf_mic_asr_offline" type="node_feedback" name="node_feedback" output="screen">
  	<param name="audio_path" type="string" value="$(find xf_mic_asr_offline)"/>  
  </node>
 
</launch>
